example/main.go

```diff
@@ -60,11 +61,11 @@ func main() {
 
 	result, err := svc.CreateMultipartUpload(input)
 	if err != nil {
-		log.ErrorC("error creating mpu", err, nil)
+		log.Event(context.TODO(), "error creating mpu", log.ERROR)
 		return
 	}
 
-	log.Debug("created multi part upload", nil)
+	log.Event(context.TODO(), "created multi part upload", log.INFO, nil)
```

Comment:
```suggestion
	log.Info(context.TODO(), "created multi part upload")
```

Reply:
and similar throughout - I'm surprised v2 allows log.Event

Reply:
This has now been amended, I ran the upgrade script in log.go and this is what it allowed.

Reply:
I think that upgrades from go-ns to log v1

Reply:
The reference in this file for the log.go library is `github.com/ONSdigital/log.go/v2/log`